google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -338,4 +343,24 @@ public StatementResult statementAbortBatch() {
     getConnection().abortBatch();
     return noResult(ABORT_BATCH);
   }
+
+  @Override
+  public StatementResult statementSetRPCPriority(Priority priority) {
+    RpcPriority value = null;
+    if (priority != null && priority != Priority.PRIORITY_UNSPECIFIED) {
+      value = RpcPriority.values()[priority.ordinal() - 1];
```

Comment:
This worries me a bit. If the order changes / a new value is added in the priority that changes the ordinal this would break without us noticing it.

Could you use a `Map<Priority, RpcPriority>` instead?